### PR TITLE
FlashLFQ initial recipe

### PR DESCRIPTION
**Pull request by jj-umn**
Installs FlashLFQ release 0.1.99
https://github.com/smith-chem-wisc/FlashLFQ

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/flashlfq/elements.dat [hunk 1]
```diff
@@ -0,0 +1,2878 @@
+<html>
+<head>
```
Comment (bgruening) [inline recipes/flashlfq/elements.dat]: Why is this file needed here @jj-umn? Can we store it on cargo-port?

Comment (jj-umn) [reply recipes/flashlfq/elements.dat]: @bgruening The FlashLFQ requires that file and expects it in the same dir as the executable.  I asked the code developer to include it in the distribution for the next release.
